automation/vars/uni02beta.yaml

```diff
@@ -0,0 +1,43 @@
+---
+vas:
+  uni02beta:
+    stages:
+      - path: examples/dt/uni02beta/control-plane/nncp
+        wait_conditions:
+          - >-
+            oc -n openstack wait nncp
+            -l osp/nncm-config-type=standard
+            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+            --timeout=60s
+          # - >-
+          #   oc label node master-0 openstack.org/cinder-lvm=''
+        values:
+          - name: network-values
+            src_file: values.yaml
+        build_output: nncp.yaml
+
+      - path: examples/dt/uni02beta/control-plane
+        wait_conditions:
+          - >-
+            oc -n openstack wait openstackcontrolplane
+            controlplane
+            --for condition=Ready
+            --timeout=30m
+        values:
+          - name: network-values
+            src_file: nncp/values.yaml
+          - name: service-values
+            src_file: service-values.yaml
+        build_output: control-plane.yaml
+
+      - path: examples/dt/uni02beta
+        wait_conditions:
+          - >-
+            oc -n openstack wait openstackdataplanedeployment
+            edpm-deployment
+            --for condition=Ready
+            --timeout=1200s
+        values:
+          - name: edpm-values
+            src_file: values.yaml
+        build_output: edpm.yaml
```

Comment:
Can we break them into `nodeset` and `deployment`?

Reply:
Yes, splitting the data plane steps should preferably be done before we merge this.  @dpinhas FYI

Reply:
@abays Will be done next iteration, uni02beta doesn't have the post-ceph that is known to impact the success of it.